src/olympia/abuse/cinder.py

```diff
@@ -171,3 +172,26 @@ def get_context(self):
                 for author in authors
             ],
         }
+
+
+class CinderAddonByReviewers(CinderAddon):
+    # This queue is not monitored on cinder - reports are resolved via AMO instead
+    queue = 'amo-addon-infringement'
+
+    def flag_for_human_review(self, appeal=False):
+        from olympia.reviewers.models import NeedsHumanReview
+
+        reason = (
+            NeedsHumanReview.REASON_ABUSE_ADDON_VIOLATION_APPEAL
+            if appeal
+            else NeedsHumanReview.REASON_ABUSE_ADDON_VIOLATION
+        )
+        self.addon.set_needs_human_review_on_latest_versions(reason=reason)
```

Comment:
Like https://github.com/mozilla/addons-server/pull/21376, this could be the wrong versions...